miden/src/cli/run.rs

```diff
@@ -64,7 +64,7 @@ impl RunCmd {
 
         // execute program and generate outputs
         let trace = processor::execute(&program, stack_inputs, advice_provider, execution_options)
-            .map_err(|err| format!("Failed to generate exection trace = {:?}", err))?;
+            .map_err(|err| format!("Failed to generate exection trace = {:?}", err.to_string()))?;
```

Comment:
Is this required?

Reply:
The idea was to show not the name of an error, but an error message it contains, since now if an error occurred we show something like that: `Failed to generate execution trace = CycleLimitExceeded(2000)`, instead of showing a message: `Failed to generate execution trace = "Exceeded the allowed number of cycles which equals 2000"`. Or was it done intentionally?

Reply:
Is it possible to do `format!("Failed to generate execution trace = {err}"` here?

Also, seems like "execution" is currently misspelled.

Reply:
Indeed, it works even better. Not sure why I even tried to use debug print since we have implemented `Display` trait for the `ExecutionError`. Thank you!